_posts/2020-09-11-micro-1-0-0-released/index.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: "Snowplow Micro 1.0.0 released"
+title-short: Snowplow Micro 1.0.0
+tags: [snowplow, micro]
+author: Ian
+category: Releases
+permalink: /blog/2020/09/11/snowplow-micro-1-0-0-released/
+discourse: false
+---
+
+We are pleased to announce the v1.0.0 release of [Snowplow Micro](https://github.com/snowplow-incubator/snowplow-micro#snowplow-micro).
+
+Snowplow Micro is built to enable companies running Snowplow to build automated test suites to ensure that new releases of their websites, mobile apps and server-side applications do not break tracking / Snowplow data collection.
+
+Snowplow Micro is a very small version of a full Snowplow data collection pipeline: small enough that it can be launched by a test suite. Events can be recorded into Snowplow Micro just as they can a full Snowplow pipeline. Micro then exposes an API that can be queried to understand:
+
+* How many events have been received?
+* How many of them were successfully processed vs ended up as "bad" (e.g. because the events failed validation against the corresponding schemas in the Iglu Schema Registry)
+* For any events that have successfully processed, what type of events they are, what fields have been recorded etc.
+* For any events that have not been successfully processed, what errors were generated on processing the events. (So these can be surfaced back via the test suite.)
+* This means companies can build automated test suites to ensure that specific events in an application generate specific events that are successfully processed by Snowplow.
+
+For examples of how to use Snowplow Micro, take a look at the 'How to' guides for [Cypress](https://snowplowanalytics.com/blog/2020/08/07/testing-tracking-with-micro-and-cypress/) and [Nightwatch](https://snowplowanalytics.com/blog/2020/08/07/testing-tracking-with-micro-and-nightwatch/) as well as the [examples repository](https://github.com/snowplow-incubator/snowplow-micro-examples#snowplow-micro-examples). 
+
+
+Read on below for:
+
+1. [Breaking changes since version 0.1.0](#breaking-changes)
```

Comment:
I think this should be

1. New features
2. Breaking changes
etc

New features should describe how you now have access to the post-enrich canonical event, which is much more helpful than just the raw event.  We can turn this into a good thing, not just an apology for a breaking change.

Breaking changes then says that the field names have changed as a result of adding the new feature.